### PR TITLE
Support multihash

### PR DESCRIPTION
**Pull request by billPortalNetwork**

### CODE REVIEW

**Comment by PhyrexTsai** [comment]
Please reference "issue"

File: example/ethereumClassic/getMultihash/multihash.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const EthereumClassic = require("../../../lib/BNS").EthereumClassic
+const ethereumClassic = new EthereumClassic({
+	restURL: 'https://etc-parity.0xinfra.com',
+	networkId: '3'
```
Comment (PhyrexTsai) [inline example/ethereumClassic/getMultihash/multihash.js]:
```
networkId: '1'
```

**Comment by billPortalNetwork** [comment]
Support multihash #1 #2 #3 #4 #5 #6

File: src/ICONNameService.js [hunk 2]
```diff
@@ -1,18 +1,44 @@
...
+  getAddress,
+  getContent,
+  getMultihash
+} from "./helper/ins/resolverService";
 
 class ICONNameService {
   // Provider URL
-  constructor(restURL) {
-    this.restURL = restURL
+  constructor(restURL, networkId, walletAddr) {
```
Comment (PhyrexTsai) [inline src/ICONNameService.js]: constructor(restURL, networkId, walletAddr = '<ICON_ADDRESS>')

Comment (billPortalNetwork) [reply src/ICONNameService.js]: In `BNS.js`, wallet address is initialized with default value if user doesn't provide
```js
if (config && config.walletAddr && config.walletAddr !== "") {
      this.walletAddr = config.walletAddr;
} else {
      this.walletAddr = "hxbe258ceb872e08851f1f59694dac2558708ece11";
}
```

**Pull request merged**